superx/information_extractors/item_info_extractor.py

```diff
@@ -59,9 +62,9 @@ def run_info_extractor(self):
                 xml_info_list = self.extract_information_from_parsed_xml(info_child_node)
                 if branch_id == '86' and self.current_super['store_name'] == 'victory':
                     continue
-
-                product_info_list, branch_price_list = self.fill_product_and_branch_price_tables(xml_info_list,
-                                                                                                 branch_id)
+                product_info_list, branch_price_list = self.fill_product_and_branch_price_tables(
+                    xml_info_list,
+                    branch_id)
```

Comment:
is this neccessary? It didnt exceed the line length limit and it is more readble.

Reply:
if it didnt exceed the line limit so no problem- i only changed what it said